### ISSUE TITLE
Allow exporting the scene via File -> Export

### ISSUE THREAD

**Issue by dictoon** [appleseedhq/appleseed-max#152]

**Comment by usakhelo** [comment]
After reading recent discussion I think we need File->Import feature request, what do you think @dictoon ? It's totally possible I think, may be little time consuming.

**Comment by dictoon** [comment]
Yes, it's certainly a feature we'd like to have in all integrations.

**Comment by usakhelo** [comment]
Just adding this to remember - need to implement Export Selected to support Archive Assemblies
